lib/gsheets.js

```diff
@@ -0,0 +1,85 @@
+const router = require('express').Router()
+const gsheets = require('gsheets')
+const { utcTimeParse, timeParse } = require('./formats')
+const slugify = require('./slugify')
+
+const dateParse = utcTimeParse('%x') // %x - the locale’s date
+const dateTimeParse = timeParse('%x %H:%M')
+
+const mapping = {
+  '12oN7o21txZKxTH_RypfWLLeXFfZE2P-Yu5kg-VoHmfw': 'faqs',
+  '1Y0vFkykAHuaGzkI5PEirnoGVmAREYr9RSUcFY7_okHk': 'updates',
+  '1rktqc3xhluZLH6OrdmY456LaiCOk0L2tMD5ZjwG89lo': 'events'
+}
```

Comment:
This id's here are just ugly, shame on me.
create a env var
`GSHEETS={id1: faqs,id2:updates}`
then use it like this:
```
const { GSHEETS } = process.env
const mapping = JSON.parse(GSHEETS) <- it's desired that this explodes on app start if the json is invalid.
```